app/controllers/user/users_controller.rb

```diff
@@ -33,11 +33,7 @@ def orders
   end
 
   private
-  def user_params
```

Comment:
A registered user has no need for this method as the one below encapsulates its entire functionality plus it also has special checks for empty passwords. I would recommend `user_params` being kept in the `app/controllers/user_controller` only which is where we need to verify the initial registration information passed in.